index.html

```diff
@@ -47,6 +49,59 @@ <h1 class="header-main__title title">Saving the world and growing crops</h1>
             </div>
         </div>
     </div>
+    <section class="partners">
+        <div class="container">
+            <h2>Used by the world's most innovative companies</h2>
+            <ul class="partners-row">
+                <li>
+                    <a href="#">
+                        <img src="./src/partners/partner_1.svg" alt="partner brand">
```

Comment:
Do we need this `/` in the path to images?

```suggestion
                        <img src="src/partners/partner_1.svg" alt="partner brand">
```

Reply:
And in another images too

Reply:
And delete `/` in this path `<script src="/scripts/main.js"></script>`